pykrige/lib/lapack.pyx

```diff
@@ -1,20 +1,31 @@
-# cython: profile=False
-# cython: boundscheck=False
-# cython: wraparound=False
-# cython: cdivision=True
+#cython: language_level=3, boundscheck=False, wraparound=False, cdivision=True
+# -*- coding: utf-8 -*-
 import numpy as np
 cimport numpy as np
 import scipy.linalg.blas
 import scipy.linalg.lapack
-from cpython.cobject cimport PyCObject_AsVoidPtr # only works with python 2, use capsules for python 3
```

Comment:
This whole file can probably be removed in favour of using scipy BLAS API from Cython directly.

Reply:
OK, I'll have a look at it.

Reply:
Done.